### PR TITLE
Add WordPress Playground blueprint

### PR DESCRIPTION
**Pull request by chriszarate**
This adds a working blueprint for WordPress Playground. This is not an alternative dev environment, since Playground still does not support cURL transport, object cache, etc. But it is a useful way to explore the capabilities of the plugin.

Try it with this link, which points to the `blueprint.json` in this PR:

https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/Automattic/remote-data-blocks/8110ff9b3ed36823935bcfccabf0510c9a665051/blueprint.json

It works via #75, which ships a static example API that does not rely on HTTP transport. A single Remote Data Block is available, "Conference Event":

https://github.com/user-attachments/assets/f69787d0-304b-4c7f-98ef-53153486a2f0

### CODE REVIEW

File: blueprint.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "steps": [
+    {
+      "step": "installPlugin",
+      "options": {
+        "activate": true
+      },
+      "pluginZipFile": {
+        "caption": "Installing Remote Data Blocks",
+        "resource": "url",
+        "url": "https://github-proxy.com/proxy/?repo=Automattic/remote-data-blocks&release=ae7f58e&asset=remote-data-blocks.zip"
```
Comment (mhsdef) [inline blueprint.json:35]:
```suggestion
        "url": "https://github-proxy.com/proxy/?repo=Automattic/remote-data-blocks&release=latest&asset=remote-data-blocks.zip"
```

Comment (chriszarate) [reply blueprint.json:35]: I wish, doesn't work

**Comment by mhsdef** [comment]
maybe worth tossing a step to set the site title to something non-generic? (in case of multiple open windows, for example)

```
{
      "step": "setSiteOptions",
      "options": {
        "blogname": "Remote Data Blocks"
      }
}
```

File: blueprint.json (hunk 1 above)
Comment (mhsdef) [reply blueprint.json:35]: Hmm... thought @stoph was using that somewhere.

Comment (mehmoodak) [reply blueprint.json:35]: It's working for me, able to download the zip file with `latest` in the url.

Comment (chriszarate) [reply blueprint.json:35]: @mehmoodak But is the downloaded file non-empty or unzippable? I think @stoph was working on a fix.

Comment (mehmoodak) [reply blueprint.json:35]: It's empty 😓

**Comment by jarekmorawski** [comment]
Thanks for making this happen! I can't seem to get the resources to load; it's stuck at `Loading...`. Is that expected?

image

**Comment by mehmoodak** [comment]
@jarekmorawski is this happening on a particular remote block or all remote blocks?

**Comment by chriszarate** [comment]
I synced quickly with Jarek, the issue happens when editing content via the Site Editor as opposed to the regular post editor. Will investigate

**Pull request merged**